winui/Kanban/cards.md

```diff
@@ -0,0 +1,161 @@
+---
+layout: post
+title: Cards in WinUI Kanban control | Syncfusion
+description: This section describes about cards in Syncfusion WinUI Kanban (SfKanban) control, its elements and more.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+
+# Cards in WinUI Kanban Control
+
+Cards in the WinUI Kanban Control provide a visual representation of tasks as they move through different stages of a workflow. Each card corresponds to a task and displays key information, such as the title, description, tags, and status, helping users track progress and manage tasks effectively.
+
+The appearance and content of the cards can be customized using the below properties of `KanbanModel`.
+
+* `Title` - Gets or sets the string value used to display the header of a card in the KanbanModel.
```

Comment:
- **Title** - Sets the title of the card.
- **Description** - Sets the description text of the card.
- **Category** - Sets the category of the card, determining which column it will be placed in.
- **Assignee** - Defines the assignee associated with the card in the KanbanModel.
- **Image** - Sets the image for the card, which is displayed on the right side in the default card template.
- **IndicatorColorKey** - Specifies the indicator color for the card, commonly used for categorization or prioritization based on key values.
- **Id** - Sets the unique ID of the card.
- **Tags** - Specifies the tags for the card, displayed at the bottom of the default card template.

Reply:
Changed.